org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,211 @@
+{
+    "app-id": "org.freedesktop.fwupd",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "branch": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/libexec/fwupd/fwupdtool",
+    "tags": [],
+    "finish-args": [
+        "--device=all",
+        "--filesystem=/boot",
```

Comment:
Does this require running as root, or does it just read from here?

Reply:
Yeah, I guess might be the main problem with it, root access requiremnet.

Reply:
Yes, it does need root to set things like efivars.

Reply:
Hmm, thats a bit nasty as we have a bad habit of users always running `flatpak run` as root and causing havok. I really don't want any root applications flatpaked personally.

Reply:
This isn't for typical users. End users are just going to have to wait for their distro to update the system version of fwupd. This flatpak would just be for OEMs like Dell to use internally whilst finalizing hardware versions, without getting people to run fwupd from git master. That said, if you run this flatpak as non-root you can still update all the USB and HID devices just fine; it's just the efi-based ones we have problems with. Running flatpak with root isn't ideal and I really want to find a better solution in the long run.